Tests iOS/MyList/ArchiveFiltersTests.swift

```diff
@@ -104,4 +104,23 @@ class ArchiveFiltersTests: XCTestCase {
         app.myListView.selectedTagChip(for: "not tagged").buttons.element(boundBy: 0).tap()
         XCTAssertEqual(app.myListView.wait().itemCells.count, 2)
     }
+
```

Comment:
I don't think the test above is testing the name of the test.

Reply:
Good observation! Some background on it:
The naming was to be consistent with "Favorites" and "All" test names and was not created with the idea of only testing `not tagged`, but any tag value. In general, wanted to test the functionality of the `Tagged` filter. I just decided to choose `not tagged` at that time, but we could have implemented using `tag 0` too. For UI tests, they take longer to run than Unit tests typically so there's a balance in covering all cases and in this case just wanted to test generally that filtering works from UI perspective. However, open to changing the name if that makes more sense. If no differences besides the name, then we should only keep one of the test. Hope that clarifies the reasoning :)

Reply:
@timc-mozilla Let's keep one of the tests so we're not spending time to run dup tests, else ready for approval!